### ISSUE TITLE
cue/format: attempts to align struct argument fields goes wrong

### ISSUE THREAD

**Issue by myitcv** [cue-lang/cue#1546]
### What version of CUE are you using (`cue version`)?

<pre>
$ cue version
cue version +a629625c linux/arm64
</pre>

### Does this issue reproduce with the latest release?

Yes

### What did you do?

I'm not clear what heuristic we should opt for when it comes to aligning struct fields where the structs are arguments to builtins, but the following definitely doesn't look right 😄 

```
cp x.cue x.cue.golden
exec cue fmt x.cue
cmp x.cue x.cue.golden

-- x.cue --
r1: blah("a: 2\n---\na: 4", {a: <3}, {a: 5})
t9: blah([{a: 1}, {b: int | *2}])
```

### What did you expect to see?

Passing test. Probably not worth trying to align the fields of structs that are arguments to builtins? 

### What did you see instead?

```
> cp x.cue x.cue.golden
> exec cue fmt x.cue
> cmp x.cue x.cue.golden
--- x.cue
+++ x.cue.golden
@@ -1,2 +1,2 @@
 r1: blah("a: 2\n---\na: 4", {a: <3}, {a: 5})
-t9: blah([{a:                   1}, {b:  int | *2}])
+t9: blah([{a: 1}, {b: int | *2}])

FAIL: /tmp/testscript825737010/repro.txt/script.txt:3: x.cue and x.cue.golden differ
error running repro.txt in /tmp/testscript825737010/repro.txt
```

**Comment by NoamTD** [comment]
@mvdan this test passes, I believe this was fixed in 01772341

**Comment by mvdan** [comment]
Well spotted!
